packages/mattermost-redux/src/actions/channels.ts

```diff
@@ -474,10 +472,8 @@ export function fetchMyChannelsAndMembers(teamId: string): ActionFunc {
         const state = getState();
         const shouldFetchArchived = isMinimumServerVersion(getServerVersion(state), 5, 21);
         try {
-            const channelRequest = Client4.getMyChannels(teamId, shouldFetchArchived);
-            const memberRequest = Client4.getMyChannelMembers(teamId);
-            channels = await channelRequest;
-            channelMembers = await memberRequest;
+            channels = await Client4.getMyChannels(teamId, shouldFetchArchived);
```

Comment:
This is a slight change in behaviour since we'd now be waiting for channels to be received before requesting channel members. Is that a change we actually want to make?

Reply:
Lets do promise.all here instead